lib/graphql/execution/interpreter/runtime.rb

```diff
@@ -8,6 +9,8 @@ class Interpreter
       #
       # @api private
       class Runtime
+        include GraphQL::Schema::Member::HasArguments
```

Comment:
`Member::HasArguments` is meant to contain behaviors that are shared between several parts of a GraphQL schema, so I don't think it's a good fit for the Runtime class. (For example, `HasArguments` implements the `argument(...)` configuration DSL!)

Reply:
Thanks. I think I can redefine the `load_application_object` method with new arguments in another new module.
How do you think?

Reply:
srry We have GraphQL::Schema::Member::HasArguments::ArgumentObjectLoader
I should extend this!